lib/src/postgresql_codec.dart

```diff
@@ -153,7 +155,7 @@ abstract class PostgreSQLCodec {
       }
 
       String val = value;
-      outBuffer = UTF8.encode(val);
+      outBuffer = convert.utf8.encode(val);
```

Comment:
would it make more sense to drop the convert namespace, such that when Dart 2 is made available, there is no need to change any code other than drop this transitionary package?

Reply:
in certain cases, this is possible, but not everywhere: e.g. when a file uses a type from `dart:convert` (e.g. `Converter`) or when it is referencing a const from `dart:core`. I've kept the prefix everywhere for consistency, but can remove in the first cases if needed.